recipes/PyALE/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "PyALE" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyALE-{{ version }}.tar.gz
+  sha256: 9053617f54b53b4d5205585eab3ce86f210bd3d1e2c5aac4305edc0455b5dc0b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - pandas
+    - matplotlib-base
+    - scikit-learn
+    - scipy
+    - statsmodels
+
+test:
+  imports:
+    - PyALE
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/DanaJomar/PyALE
+  summary: ALE plots with python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - https://github.com/DanaJomar
```

Comment:
It should be just the handle, not the URL.

Reply:
Done!